pandas/io/excel/_util.py

```diff
@@ -12,10 +19,15 @@
     is_list_like,
 )
 
-_writers: MutableMapping[str, str] = {}
+if TYPE_CHECKING:
+    from pandas.io.excel._base import ExcelWriter
 
+    ExcelWriter_t = Type[ExcelWriter]
```

Comment:
Can probably use `type[ExcelWriter]`

Reply:
Thx, did not know that

Reply:
PEP 585 has a nice overview list: https://www.python.org/dev/peps/pep-0585/#implementation

Reply:
Thx, this is handy